runtime/doc/helphelp.txt

```diff
@@ -433,17 +433,27 @@ To quote a block of ex-commands verbatim, place a greater than (>) character
 at the end of the line before the block and a less than (<) character as the
 first non-blank on a line following the block.  Any line starting in column 1
 also implicitly stops the block of ex-commands before it.  E.g. >
-    function Example_Func()
-	echo "Example"
-    endfunction
+	function Example_Func()
+	  echo "Example"
+	endfunction
```

Comment:
why the indentation change?

Reply:
I have read the other example indentation.

It seems indented by tab.